README.md

```diff
@@ -848,6 +850,54 @@ const Screen = () => (
 )
 ```
 
+### 7.5 Styled Component Naming Convention
+
+Use PascalCase as a convention in styled-components
+
+**✅ Good:**
+
+```js
+export const CustomText = styled.p`
+  color: 'red'
+`
+```
+
+**❌ Bad:**
+
+```js
+export const customText = styled.p`
+  color: 'red'
+`
+```
+
+### 7.6 Using Styled Component in React Components
+
+Import Styled Component as `S`
+
+**✅ Good:**
+
+```tsx
+import * as S from './styles'
+
+const MyComponent = () => (
+  <S.CustomText>
+    text example
+  </S.CustomText>
+)
+```
+
+**❌ Bad:**
+
+```tsx
+import * as Style from './styles'
+
+const MyComponent = () => (
+  <Style.CustomText>
+    text example
+  </Style.CustomText>
+)
```

Comment:
Maybe we could add a more conventional case. What do you think?
```suggestion
import { CustomText } from './styles'

const MyComponent = () => (
  <CustomText>
    text example
  </CustomText>
)
```

Reply:
Perfect.
Done: 495454127df8b725aa027146b5d623774cb6245c